torchvision/transforms/v2/_misc.py

```diff
@@ -284,7 +284,9 @@ def forward(self, *inputs: Any) -> Any:
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
-        if (inpt is not None and inpt is params["labels"]) or isinstance(inpt, datapoints.BoundingBox):
+        if (inpt is not None and inpt is params["labels"]) or isinstance(
+            inpt, (datapoints.BoundingBox, datapoints.Mask)
```

Comment:
One reason why we used the key `"is_valid"` or the like previously, is because while `"mask"` is correct, it somewhat overlaps with the `datapoints.Mask` inputs. And since `params["mask"]` acts only on the batch dimensions, we wanted to avoid this link over the terminology.

Reply:
I know this is pedantic, but `"valid_indices"` is not accurate, since we are still talking about a bool mask rather than integer indices.

Reply:
feel free to change to other names